### ISSUE TITLE
problems in installation

### ISSUE THREAD

**Issue by vaxin** [davidsandberg/facenet#222]
after installation, the python site-packages directory appears "src" "test" dirs, so the "import facenet" does not work. Only "from src import facenet" works.

**Comment by xf4fresh** [comment]
@vaxin 
I encountered the same problem. I think this is related to the environment or the system, you can choose the code to run the normal.

**Comment by fedyunin** [comment]
https://github.com/davidsandberg/facenet/wiki/Classifier-training-of-inception-resnet-v1#3-set-the-python-paths

**Comment by hamstah** [comment]
The package should either be valid or not be an option.

**Comment by hamstah** [comment]
I think this fixes the package issue:
* rename `src` to `facenet`
* Edit setup.py
```
#!/usr/bin/env python

from distutils.core import setup

setup(name='facenet',
      version='0.1',
      description='Face Recognition using Tensorflow',
      author='David Sandberg',
      author_email='david.o.sandberg@gmail.com',
      url='https://github.com/davidsandberg/facenet/',
      packages=['facenet','facenet.align','facenet.models'],
     )
```
* `pip wheel .`
* `pip install facenet-*.whl`
* `from facenet.align import detect_face`

**Comment by hamstah** [comment]
@davidsandberg It would be great if you could upload it to pypi so we can install it without having to git clone and mess with the path

**Comment by davidsandberg** [comment]
@hamstah I would rather not upload it to pypi since it would send the signal that this repo is a finalized application rather than a framework to simplify your own research in the face recognition area.

**Comment by hamstah** [comment]
I see. Would you be open to make the changes to the setup file to make it
easier to package it?
I can make a or if you're ok with the idea.

Thanks!

**Comment by davidsandberg** [comment]
If you make a PR on setup.py I will be happy to merge it.

**Comment by hamstah** [comment]
Great. to get it to work, src has to be renamed to facenet. It's a bigger
change than just the setup.py. I might need your help to double check
documentation etc to make sure all paths are updated.

Let me know if you're ok with that change, I understand if it's too big for
now.

**Comment by eacosta1976** [comment]
Hi my name is Edgar Acosta.  I want to run Facenet,  detect face, the name of the person, and emotions.  I download TensorFlow (Using container), I have python 2.7,  and also iPython and Jupyter as my interactive editor.     I need some advice: 

   To clone Facenet: Do I need to clone inside Tensor Flow directory?. 
                                  Do you believe my setup is OK, or do I need to install something else.?
                                  Is there any instructions to make it up and running with my WEB Cam?
     
 Thank you very basic questions but I am very new to this Fascinating field.

  My email address is : eacosta1976@gmail.com

**Comment by zixia** [comment]
> I would rather not upload it to PyPI since it would send the signal that this repo is a finalized application rather than a framework to simplify your own research in the face recognition area.

@davidsandberg Could we add a WARNING message in PyPI package to let users know the package is not finalized but only a package installer for convenience?

**Comment by Yashkhandhediya** [comment]
Hi my name is Yash Khandehediya. I've cloned facenet code and tried to run it. Actually I am executing this code in small snippets for better understanding of code but problem I encountered is that, at this part of the code
""
    nrof_preprocess_threads = 4
    images_and_labels = []
    for _ in range(nrof_preprocess_threads):
        filenames, label = input_queue.dequeue()
        images = []
        for filename in tf.unstack(filenames):
            file_contents = tf.read_file(filename)
            image = tf.image.decode_image(file_contents, channels=3)
            if args.random_rotate:
                image = tf.py_func(facenet.random_rotate_image, [image], tf.uint8)
            if args.random_crop:
                image = tf.random_crop(image, [args.image_size, args.image_size, 3])
            else:
                image = tf.image.resize_image_with_crop_or_pad(image, args.image_size, args.image_size)
            if args.random_flip:
                image = tf.image.random_flip_left_right(image)
    
            #pylint: disable=no-member
            image.set_shape((args.image_size, args.image_size, 3))
            images.append(tf.image.per_image_standardization(image))
        images_and_labels.append([images, label])
    
""
my system hangs and shows no response for hours and days. So I want to ask that do loops make tensorflow slower? if yes or not what can be the solution to this problem. I also tried by using small batch size and less epoch size. 

Thank you.

my email id is: yashhkhandhediya@gmail.com

**Comment by xixikaikai** [comment]
can I now use pip install to use facenet?
pip install facenet?
